Common subdirectories: iPodder-linux/BitTorrent and iPodder-linux-er/BitTorrent
Common subdirectories: iPodder-linux/Resources and iPodder-linux-er/Resources
Common subdirectories: iPodder-linux/build and iPodder-linux-er/build
diff -U 2 iPodder-linux/erik_reload_module.py iPodder-linux-er/erik_reload_module.py
--- iPodder-linux/erik_reload_module.py	2005-02-16 00:35:54.000000000 -0500
+++ iPodder-linux-er/erik_reload_module.py	2005-06-19 19:19:54.000000000 -0400
@@ -56,4 +56,7 @@
     fds = self.ipodder.feeds
 
+	import wxversion
+	wxversion.select("2.6-unicode")
+
     import wx
     count = 0
Common subdirectories: iPodder-linux/gui and iPodder-linux-er/gui
diff -U 2 iPodder-linux/iPodderGui.py iPodder-linux-er/iPodderGui.py
--- iPodder-linux/iPodderGui.py	2005-04-16 01:04:07.000000000 -0400
+++ iPodder-linux-er/iPodderGui.py	2005-06-19 19:19:06.000000000 -0400
@@ -15,4 +15,6 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
+import wxversion
+wxversion.select("2.6-unicode")
 
 import wx
diff -U 2 iPodder-linux/iPodderWindows.py iPodder-linux-er/iPodderWindows.py
--- iPodder-linux/iPodderWindows.py	2005-04-16 09:59:29.000000000 -0400
+++ iPodder-linux-er/iPodderWindows.py	2005-06-19 19:20:16.000000000 -0400
@@ -1,2 +1,4 @@
+import wxversion
+wxversion.select("2.6-unicode")
 
 import wx
Common subdirectories: iPodder-linux/icons_status and iPodder-linux-er/icons_status
Common subdirectories: iPodder-linux/images and iPodder-linux-er/images
Common subdirectories: iPodder-linux/ipodder and iPodder-linux-er/ipodder
Common subdirectories: iPodder-linux/localization and iPodder-linux-er/localization
Common subdirectories: iPodder-linux/mp3 and iPodder-linux-er/mp3
Common subdirectories: iPodder-linux/osx_packaging and iPodder-linux-er/osx_packaging
Common subdirectories: iPodder-linux/sandbox and iPodder-linux-er/sandbox
Common subdirectories: iPodder-linux/win32 and iPodder-linux-er/win32
